### DISCUSSION TITLE
——— [Sep 24]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3900] [Discord Threads]
> 🙋‍♂️ **Jess (she/her)**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Braks**  
#### I didn't see a particular need for pinia because provide/inject does the trick 😅   
#### And for a "global" state we just insert an object into the NuxtApp so it's accessible using `useNuxtApp`    
####    
#### Is there a particular benefit in using pinia vs provide/inject I might not be aware of?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1020524944389574778)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
